tests/test_clientmanager.py

```diff
@@ -28,7 +29,7 @@ def test_deserialize_on_creation():
 
 
 def test_serialize_periodically():
-    _t_config_hook({"serialize_period_seconds": 0.1})
+    conf.override({"serialize_period_seconds": 0.1, "serialize_periodically": True})
```

Comment:
Suggestion: refer to the method's __name__ property here, so that if we change the name, a refactor tool will change the name here too.